### DISCUSSION TITLE
Resume Bundle x ATS Scoring

### DISCUSSION THREAD

**Discussion by Nihal4777** [recodehive/resume-pitch#5] [Q&A] [answered]
[Nihal_Rajpal_Resume.pdf](https://github.com/user-attachments/files/16558262/Nihal_Rajpal_Resume.pdf)

**Comment by sanjay-kv** [comment]
Hi Nihal saw this now, will reply you by EOD @Nihal4777

**Comment by sanjay-kv** [comment]
[nihal rev.pdf](https://github.com/user-attachments/files/16580015/nihal.rev.pdf)
[ATS Version of CV (9).docx](https://github.com/user-attachments/files/16580016/ATS.Version.of.CV.9.docx)

**Reply by sanjay-kv** [reply, marked as answer]
https://www.loom.com/share/e1a4a989dbd4438d8e91cdc09f1ea718?sid=43a9e170-4e7f-4634-abac-0193b31692be

@Nihal4777

**Reply by Nihal4777** [reply]
Okay Sir, 
Thank you very much for your valuable suggestions. Will definitely implement them.

**Reply by Nihal4777** [reply]
Hi Sir, I just had few doubts.
 
It says email & LinkedIn not detected although both are added. What can i do to solve this? 
![image](https://github.com/user-attachments/assets/fe3817f6-417b-4bce-af75-927d81e6e6f1)

**Reply by sanjay-kv** [reply]
usually ATS software cant fetch hyperlink. if you have short linkedin URL you can use that else can skip this .

**Comment by Anishkagupta04** [comment]
hello this side Anishka Gupta this is my resume attached below
[ANISHKA_RESUME (5).pdf](https://github.com/user-attachments/files/17005168/ANISHKA_RESUME.5.pdf)
